examples/v2/sqladmin/README.md

```diff
@@ -4,16 +4,27 @@
 
 This is a [Google Cloud Deployment
 Manager](https://cloud.google.com/deployment-manager/overview) template that
-deploys a Cloud SQL instance and database. We also use a DM `action` to
-delete the root user. Actions are a currently undocumented feature for
+deploys a Cloud SQL Master instance and N read-replicas. We also use a DM `action` to
+add users to the instance and delete the root@% user. Actions are a currently undocumented feature for
 calling arbitrary APIs.
 
+> Note, CloudSQL does not allow parallel updates to the User table
+
+## Prerequsites
+
+You need to grant 
+
+- [roles/cloudsql.admin](https://cloud.google.com/iam/docs/understanding-roles#sql_name_short_roles)  role
+to the service account DM uses (```projectNumber@cloudservices.gserviceaccount.com```)
+
 ## Deploy the template
 
 Use `sqladmin.yaml` to deploy this example template. When ready, deploy
 with the following command:
 
+```
     gcloud deployment-manager deployments create my-database --config sqladmin.yaml
```

Comment:
If you're going to wrap it in triple ticks, I would remove the 4 preceding spaces.

Reply:
done;